vote/src/vote_account.rs

```diff
@@ -550,6 +627,30 @@ mod tests {
         })
     }
 
+    // The difference between this and `new_rand_vote_accounts` is that this
+    // generates fixed number of alpenglow nodes, and all vote accounts
+    // generated have non-zero stake.
+    // Also we return a default HashMap of identity account balances.
+    fn new_staked_vote_accounts<R: Rng>(
+        rng: &mut R,
+        num_nodes: usize,
+        num_alpenglow_nodes: usize,
+        stake_per_node: Option<u64>,
+    ) -> (VoteAccounts, HashMap<Pubkey, u64>) {
+        let mut vote_accounts = VoteAccounts::default();
+        let mut identity_balances = HashMap::new();
+        for index in 0..num_nodes {
+            let is_alpenglow = index < num_alpenglow_nodes;
+            let pubkey = Pubkey::new_unique();
+            let stake = stake_per_node.unwrap_or_else(|| rng.gen_range(1..997));
```

Comment:
should comment on the magic numbers or pull into a const

Reply:
Good point, changed into const.